infra/terraform/modules/data_access/github_webhooks_handler/tests/conftest.py

```diff
@@ -0,0 +1,140 @@
+from hashlib import sha1
+from moto import mock_sns
+import boto3
+import hmac
+import json
+import os
+
+import pytest
```

Comment:
So `from moto import mock_sns` is before `import pytest` because `moto` is less than `pytest` right?  Or it sorted by the name imported (e.g. `mock_sns` in this case).

Anyway, I'll sort 👍